### ISSUE TITLE
Xbox Controller Emulation

### ISSUE THREAD

**Issue by lubosz** [falkTX/qtsixa#2]
Is it possible to emulate an Xbox Controller with QtSixA?
It is not clear to me how to use profiles to achieve that.
It should not be complicated, since its just a button mapping.

It would not be neccecary to use xboxdrv

http://pingus.seul.org/~grumbel/xboxdrv/
https://github.com/Grumbel/xboxdrv

Since a lot of games use this mapping on Steam, this would be a great feature.

**Comment by cockfight** [comment]
I agree with this, it would be awesome if we could somehow map every button press to appear as an xbox controller, this is what unfortunately 90% of games expect today :(

**Comment by mdeguzis** [comment]
The only way I achieved that was with xboxdrv. Not sure how else you can do that.
